angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -105,7 +106,8 @@ export class CustomAppComponent {
             stringValue1: `new string ${tableData.length}`,
             stringValue2: `bar ${tableData.length}`,
             href: '/customapp',
-            linkLabel: 'Link'
+            linkLabel: 'Link',
+            statusCode: (Math.floor(Math.random() * 10) % 2) === 0 ? 100 : 101
```

Comment:
I don't have a strong opinion either way, but I'm curious if there's a reason to make this random rather than deterministic (i.e. `statusCode: (tableData.length % 2 === 0) ? 100 : 101`).